### PR TITLE
clean up DataInstaller

### PR DESCRIPTION
**Pull request by grimreaper**
Clean up DataInstaller:
 	- use weak types whenever possible
 	- use diamond operator
 	- use try resource closing instead of manual closing
 	- use .equals for objects
 	- move inner functions where they belong
 	- add missing end of file newlines

### CODE REVIEW

**Comment by LegacyKing** [comment]
@thpr & @jdempsey-au - how do this look?

File: code/src/java/pcgen/gui2/dialog/DataInstaller.java [hunk 1]
```diff
@@ -180,6 +274,134 @@ else if (source == installButton)
...
+		 */
+		private boolean readDataSet(File dataSet)
+		{
+			// Open the ZIP file
+			try (ZipFile in = new ZipFile(dataSet)) {
+				// Get the install file in a case insensitive manner
+				ZipEntry installEntry = null;
+				@SuppressWarnings("rawtypes")
+				Enumeration entries = in.entries();
+				while (entries.hasMoreElements()) {
```
Comment (jdempsey-au) [inline code/src/java/pcgen/gui2/dialog/DataInstaller.java]: I think you need to grab our formatting rules and apply them in your editor. Braces such as these should be on their own line.

Comment (LegacyKing) [inline code/src/java/pcgen/gui2/dialog/DataInstaller.java]: James, noob question, how would you apply these rules in say Netbeans or Eclipse IDEs?

**Comment by LegacyKing** [comment]
@grimreaper Eitan, did you get the message? Would like to get these two PRs included ASAP. Please address. Thanks!

**Comment by LegacyKing** [comment]
@grimreaper you still around?

**Comment by grimreaper** [comment]
Sorry: I've been busy and clearly don't have time to come back to this. Closing for now, but I'll keep the email around so that if I do have time I'll retry.

I hate doing drive-by pull requests without following up, but in this case the time factor kicked in.

**Pull request closed without merging**